modules/manually-create-identity-access-management.adoc

```diff
@@ -228,6 +228,24 @@ spec:
 +
 endif::ash[] 
 
+.Sample `Secret` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-image-registry
+  name: installer-cloud-credentials
+data:
+  azure_subscription_id: <base64-encrypted-azure-subscription-id>
+  azure_client_id: <base64-encrypted-azure-client-id>
+  azure_client_secret: <base64-encrypted-azure-client-secret>
+  azure_tenant_id: <base64-encrypted-azure-tenant-id>
+  azure_resource_prefix: <base64-encrypted-azure-resource-prefix>
+  azure_resourcegroup: <base64-encrypted-azure-resourcegroup>
+  azure_region: <base64-encrypted-azure-region>
+----
+
```

Comment:
Should there be a `+` after the code block? Looking at the preview, the indentation of the IMPORTANT notice and the following procedure steps doesn't seem right.

Reply:
Addressed